adding clarification to `neo4j-admin backup` command description (#1227)
neo4j/docs-operations: modules/ROOT/pages/backup-restore/online-backup.adoc

|

|--include-metadata=none\|all\|users\|roles

> |Include metadata in file. Can't be used for backing up the system database.

Author: thanks. What about:

```suggestion
|Include metadata in the file. This cannot be used for backing up the `system` database. Possible values are:
```

> |Include metadata in file. Can't be used for backing up the system database.

Reviewer: Sounds good.

- `roles` - include commands to create the roles and privileges (for both database and graph) that affect the use of the database
- `users` - include commands to create the users that can use the database and their role assignments
- `all` - include `roles` and `users` detailed above

> - `none` - don't include any metadata

Reviewer: ```suggestion
- `none` - does not include any metadata.
```

[NOTE]

> `roles` and `users` that do not have database-related privileges are not included in the backup (e.g. those with only DBMS or no privileges). It is recommended to use `SHOW USERS` and `SHOW ROLES` to get the complete list of users and roles in these situations.

Author: the lack of PUBLIC privileges being backed up was a separate bug

> `roles` and `users` which don't have any database-related privileges are not included in the backup (e.g. those with only DBMS privileges or no privileges at all). It is recommended to use `SHOW USERS` and `SHOW ROLES` to get the full list of users and roles in these situations.

Reviewer: ```suggestion
`roles` and `users` that do not have database-related privileges are not included in the backup (e.g. those with only DBMS or no privileges). It is recommended to use `SHOW USERS` and `SHOW ROLES` to get the complete list of users and roles in these situations.
```
